prov/efa/src/rxr/rxr_msg.c

```diff
@@ -154,10 +151,9 @@ int rxr_msg_select_rtm(struct rxr_ep *rxr_ep, struct rxr_tx_entry *tx_entry, int
 		return RXR_EAGER_MSGRTM_PKT + tagged;
 	}
 
-	if (use_p2p && efa_mr_is_cuda(tx_entry->desc[0]))
-		return rxr_msg_select_rtm_for_cuda(rxr_ep, tx_entry);
+	if (use_p2p && efa_mr_is_hmem(tx_entry->desc[0]))
```

Comment:
I do not think we will do runting read for habana?

Reply:
I think I could just insert the habana block before this `if`

Reply:
Hababa is not in the code base yet. so I did not include it.

After habana is added, I think it is better to handle it inside the function `select_rtm_for_hmem`.

Reply:
Hmm I think I could just insert the habana block before this `if`.

Reply:
@wzamazon That sounds good. Thank you!